package.json

```diff
@@ -12,7 +12,8 @@
   "scripts": {
     "test": "node node_modules/.bin/mocha --compilers js:babel/register --bail --timeout 3000",
     "compile": "node node_modules/.bin/babel src --out-dir lib --optional runtime",
-    "prepublish": "npm run compile"
+    "prepublish": "npm run compile",
+    "lint": "eslint ./src"
```

Comment:
Can it be this instead?

```
"lint": "node node_modules/.bin/eslint ./src"
```


That way there's no dependency on a global `eslint` executable in the user's environment.